com.unity.render-pipelines.core/Runtime/Volume/KeyframeUtility.cs

```diff
@@ -0,0 +1,294 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Reflection;
+using Unity.Collections;
+using UnityEngine.Assertions;
+
+namespace UnityEngine.Rendering
+{
+    /// <summary>
+    /// A helper function for interpolating AnimationCurves together. In general, curves can not be directly blended
```

Comment:
It would be nice if you add an example to this documentation `<example></example>`

Reply:
Sure. Adding one now.

Reply:
thanks